packages/bigtable/src/table.js

```diff
@@ -861,8 +867,11 @@ Table.prototype.mutate = function(entries, callback) {
     })
   ]);
 
-  if (isStreamMode) {
-    return stream;
+  if (!isCallbackMode) {
+    ['error', 'finish'].forEach(function(event) {
```

Comment:
No, it's okay to do both. Let's just map `finish` to `complete` to match the Operation lingo.